client/my-sites/checkout/composite-checkout/components/checkout-help-link.tsx

```diff
@@ -142,6 +144,13 @@ export default function CheckoutHelpLink() {
 	const { setShowHelpCenter } = useDataStoreDispatch( HELP_CENTER_STORE );
 	const isEnglishLocale = useIsEnglishLocale();
 
+	const cartKey = useCartKey();
+	const { responseCart } = useShoppingCart( cartKey );
+
+	const cartErrors = responseCart.messages?.errors || [];
+	const purchasesAreBlocked =
+		cartErrors.filter( ( error: ResponseCartMessage ) => error.code === 'blocked' ).length > 0;
```

Comment:
Really minor but you could also use `some` instead of `filter` and `length`.

```suggestion
		cartErrors.some( ( error: ResponseCartMessage ) => error.code === 'blocked' );
```